pkg/piecestore/rpc/client/client.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2018 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package client
+package psclient
```

Comment:
Let's move the package and rename the folder while were at it. Let's make it `pkg/piecestore/psclient` and `pkg/piecestore/psserver`. Can be a separate PR if you think this would get too confusing.